<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChiyoLoader</class>
 <widget class="QWidget" name="ChiyoLoader">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chiyo Loader</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLabel" name="loading">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>320</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="progressBarText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>320</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading plugins...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="loadingErrors">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
